collect_app/src/main/AndroidManifest.xml

```diff
@@ -93,7 +93,7 @@ the specific language governing permissions and limitations under the License.
 
         <activity
             android:name=".activities.MainMenuActivity"
-            android:configChanges="locale|orientation|screenSize" />
+            android:configChanges="locale" />
```

Comment:
Are you leaving `locale` because you think it's needed or just because it's not related to what you're doing here?